README.md

```diff
@@ -33,6 +33,21 @@ You might want to customize the workflow in the following ways:
  - **branch** You can customize the branch that is pushed to by changing `UPDATE_BRANCH`.
 
 
+## 3. Cache
+
+Currently, we cache the "sourcecred_data" folder is the root of the repository that contains
+previously extracted metadata about contributions. For most cases, this will assist to run
+the action more quickly, however if you find that a different version of `sourcecred/sourcecred:dev`
+is being used (and either outputs an error, or your contribution plot seems off) you can manually
+invalidate the cache by removing this folder entirely before running the containers:
+
+```bash
+rm -rf sourcecred_data
```

Comment:
So we should run this as part of our update script? Is there any way we can avoid running a one-off with a different script?

Reply:
I was suggesting (to the user) to add it to the recipe, either the automated.yml or pull_request.yml. Since it would be a one off failure, it would come down to a one off action... I'm not sure how else to do it. If we take some kind of hash of the content, when contributions change (but the container doesn't) it would be different. Oh - what if we kept the version of the dev container somewhere?

Reply:
@Beanow did you have any other thoughts on this?

Reply:
No, for the time being asking users to throw away their data folder is an easy approach and something I do all the time developing. In the future I believe we should move towards https://github.com/sourcecred/sourcecred/issues/1475